src/thor/bidirectional_astar.cc

```diff
@@ -529,8 +533,10 @@ BidirectionalAStar::GetBestPath(valhalla::Location& origin,
   // PathLocation using edges.front here means we are only setting the
   // heuristics to one of them alternate paths using the other correlated
   // points to may be harder to find
-  SetOrigin(graphreader, origin, forward_time_info);
-  SetDestination(graphreader, destination, reverse_time_info);
+  baldr::graph_tile_ptr origin_potential_tile = nullptr;
```

Comment:
I would rename it to `forward_tile` or `forward_tile_hint`; and the same for `destination_potential_tile`